### DISCUSSION TITLE
Voting power = 1 for all the token holders

### DISCUSSION THREAD

**Discussion by herniadlf** [snapshot-labs/snapshot#3569] [Q&A] [answered]
hey! 
Is there a way to setup or combine strategies that gives voting_power=1 for all the holders of certain erc20? I was trying to combine erc20-balanceof and math but i couldnt make it. Thx!

**Comment by samwhosung** [comment, marked as answer]
Sure, you can use the Ticket strategy alongside "Basic" voting validation. When defining the voting validation parameters, you have the option to specify the `strategies` for the tokens and the `minScore` required for voters to be eligible to vote. @herniadlf

**Reply by samwhosung** [reply]
Here is an example of the parameters of Basic validation for ERC20 token,

```json
{
  "minScore": 1,
  "strategies": [
   {
      "name": "erc20-balance-of",
      "params": {
        "address": "0x6b175474e89094c44da98b954eedeac495271d0f",
        "symbol": "DAI",
        "decimals": 18
      }
  ]
}
```

**Reply by herniadlf** [reply]
OOhh great! I found some crazy strategy alternative.. But i prefer your approach. I sahre my example for future readers:
```
{
  "symbol": "MATH",
  "operands": [
    {
              "type": "strategy",
              "strategy": {
                "name": "erc20-balance-of",
                "network": "1",
                "params": {
                  "address": <ERC20_ADDRESS>,
                  "symbol": "DAI",
                  "decimals": 18
                }
              }
            },
    {
      "type": "constant",
      "value": 1
    },
    {
      "type": "constant",
      "value": 0
    }
  ],
  "operation": "a-if-gt-b"
}
```

If the balance is greater than 0, the result is "1". And if not, the result is "0"
